source/docs/3/nearest_neighbor_analysis.rst

```diff
@@ -34,103 +39,74 @@ Procedure
 
 1. Locate the downloaded ``ne_10m_populated_places_simple.zip`` file in the :guilabel:`Browser` panel and expand it. Drag the ``ne_10m_populated_places_simple.shp`` file to the canvas.
 
-  .. image:: /static/3/nearest_neighbor_analysis/images/1.png
+  .. image:: /static/3/nearest_neighbor_analysis/images/01.png
      :align: center
 
 2. You will see a new layer ``ne_10m_populated_places_simple`` loaded in the :guilabel:`Layers` panel. This layer contains the points representing populated places. Now we will load the earthquakes layer. This layer comes as a *Tab Serepated Values (TSV)* text file. To load this file, click the :guilabel:`Open Data Source Manager` button on the :guilabel:`Data Source Toolbar`. You can also use :kbd:`Ctrl + L` keyboard shortcut.
 
-  .. image:: /static/3/nearest_neighbor_analysis/images/2.png
+  .. image:: /static/3/nearest_neighbor_analysis/images/02.png
      :align: center
 
-3. Click the :guilabel:`...` button next to :guilabel:`File name` and browse to the downloaded ``signif.txt`` file. Once loaded, the :guilabel:`File Format` and :guilabel:`Geometry Definition` fields should be auto-populated with correct values. Click :guilabel:`Add` followed by :guilabel:`Close`.
+3. In the :guilabel:`Data Source Manager` dialog box, select :guilabel:`Delimited Text`. 
 
-  .. image:: /static/3/nearest_neighbor_analysis/images/3.png
+  .. image:: /static/3/nearest_neighbor_analysis/images/03.png
      :align: center
 
-4. Zoom around and explore both the datasets. Each yellow point represents the location of a significant earthquake and each red point represents the location of a populated place. Our goal is to find out the nearest point from the populated places layer for each of the points in the earthquake layer.
+4. Click the :guilabel:`...` button next to :guilabel:`File name` and browse to the downloaded ``earthquakes-2021-11-25_13-39-30_+0530.tsv`` file. Once loaded, the :guilabel:`File Format` and :guilabel:`Geometry Definition` fields should be auto-populated with correct values. Cross verify it with the markings below if required update field acordingly. Click :guilabel:`Add` followed by :guilabel:`Close`.
```

Comment:
As in the CSV tutorial, add this sentence. Don't need a separate step but it should be mentioned

Depending upon the operating system, you may or may not view the file at the downloaded location. In File formats, switch to All files (*; *.*) to view the tsv file.

Reply:
done.